architecture/service_catalog/index.adoc

```diff
@@ -62,6 +62,13 @@ their own. For example, a service instance may be a BestDataBase instance named
 application). For example, the `my_db` service instance may be bound to the
 user's application called `my_app`.
 
+When a user makes a request to provision or deprovision a resource, the request
+is made to the service catalog which then proxies the request to the appropriate
+cluster service broker.  With some services, some operations such as provision,
+deprovision and update are expected to take some time to fulfil and are
+asynchronous (queued and run in the background).  If the cluster service broker
```

Comment:
I wonder if we even need to mention the asynchronous aspect of this here.  I would avoid the language about queuing.